walkthroughs/howto-k8s-tls-acm/README.md

```diff
@@ -0,0 +1,224 @@
+## Overview
+In this walkthrough we'll enable TLS encryption between two applications in App Mesh using private certificate from AWS Certificate Manager issued by an AWS Certificate Manager Private Certificate Authority (ACM PCA) .
+
+## Prerequisites
+[Walkthrough: App Mesh with EKS](../eks/)
+
+The manifest in this walkthrough requires [aws-app-mesh-controller-for-k8s](https://github.com/aws/aws-app-mesh-controller-for-k8s) version [>=v1.0.0](https://github.com/aws/aws-app-mesh-controller-for-k8s/releases/tag/v1.0.0). Run the following to check the version of controller you are running.
+```
+$ kubectl get deployment -n appmesh-system appmesh-controller -o json | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'|tail -n1
+```
+
+## Step 1: Setup environment
+1. Clone this repository and navigate to the walkthrough/howto-k8s-tls-acm folder, all commands will be executed from this location
+2. Your AWS account id:
+
+    export AWS_ACCOUNT_ID=<your_account_id>
+
+3. Region e.g. us-west-2
+
+    export AWS_DEFAULT_REGION=us-west-2
+
+4. ENVOY_IMAGE environment variable is set to App Mesh Envoy, see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
+
+    export ENVOY_IMAGE=...
+
+5. SERVICES_DOMAIN to be used while creating the AWS Certificate Manager Private Certificate Authority
+
+     export SERVICES_DOMAIN="howto-k8s-tls-file-based.svc.cluster.local"
+
+
+## Step 2: Create a Certificate
+
+Before we can encrypt traffic between services in the mesh, we need to generate a certificate. App Mesh currently supports certificates issued by an [ACM Private Certificate Authority](https://docs.aws.amazon.com/acm-pca/latest/userguide/PcaWelcome.html), which we'll setup in this step.
+
+Start by creating a root certificate authority (CA) in ACM.
+
+> You pay a monthly fee for the operation of each AWS Certificate Manager Private Certificate Authority until you delete it and you pay for the private certificates you issue each month. For more information, see [AWS Certificate Manager Pricing](https://aws.amazon.com/certificate-manager/pricing/).
```

Comment:
Definitely good to call this out loudly, maybe add a "NOTE" or similar to stand out even more

Reply:
Added the comment in bold and italic